parser/expr_equal.go

```diff
@@ -453,6 +453,18 @@ func equalTuple(x, y *tipe.Tuple) bool {
 	return true
 }
 
+func equalTypes(t0, t1 []tipe.Type) bool {
+	if len(t0) != len(t1) {
+		return false
+	}
+	for i := range t0 {
+		if !equalType(t0[i], t1[i]) {
+			return false
+		}
+	}
+	return true
+}
+
 func equalType(t0, t1 tipe.Type) bool {
```

Comment:
Huh. I just realized we have `type.Equal`. Why do we have this function? :-)

Reply:
I'll remove it in a later PR

Reply:
I've filed neugram/ng#88 to remember this.